tests/Custom/SelectTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
```

Comment:
There should be no Driver-specific test in this case, because `setValue` for select should behave the same way for all drivers and therefore test should go to the `Mink` main repo instead.